stable/percona-xtradb-cluster/templates/config-map_mysql-config.yaml

```diff
@@ -8,4 +8,15 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 data:
+{{- if .Values.configFiles }}
 {{ toYaml .Values.configFiles | indent 2 }}
+{{ if not (index .Values.configFiles "node.cnf") }}
+  node.cnf: |
+  {{- range .Files.Lines "files/node.cnf" }}
+  {{ . | indent 2 }}{{ end }}
+{{ end }}
```

Comment:
Why not just do this instead of using a range?

```yaml
data:
  node.cnf: |
{{ .Files.Get "files/node.cnf" | indent 4 }}
```

Reply:
I had tried that (or something like it) and it had introduced some weird indenting issues ... but trying it again just now it seems to work fine so I'll update as its much cleaner.